Tensile/Code.py

```diff
@@ -256,7 +256,7 @@ def instructions(self):
 
     if len(main_args) > 0:
       rv.addInst("s_waitcnt", *main_args, self.comment)
-      if wait_store and self.version == (10,1,0):
```

Comment:
this should be an architectural cap like HasEccHalf rather than inline version compare